internal/rbd/nodeserver.go

```diff
@@ -700,7 +725,8 @@ func (ns *NodeServer) mountVolumeToStagePath(
 	ctx context.Context,
 	req *csi.NodeStageVolumeRequest,
 	staticVol bool,
-	stagingPath, devicePath string) error {
+	stagingPath, devicePath string, fileEncryption bool,
```

Comment:
coding guidelines suggest to place new parameters of different types on the next line